src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery newBall = new Lottery();
```

Comment:
```suggestion
        Lottery lottery = new Lottery();
```
чому lottery має назву newBall ? 
це ж не кулька, лотерея створює кульки